src/interfaces/node.cpp

```diff
@@ -359,7 +359,7 @@ class NodeImpl : public Node
     {
 #ifdef ENABLE_WALLET
         std::vector<std::unique_ptr<Wallet>> wallets;
-        for (CWalletRef wallet : ::vpwallets) {
+        for (CWalletRef wallet : GetWallets()) {
```

Comment:
can you explain 800029f2de473b7d874c910162e4215d2b2c0f49?

Reply:
Just reverting some accidental changes